ios/RNPrint/RNPrint.m

```diff
@@ -17,7 +17,18 @@ - (dispatch_queue_t)methodQueue
                   resolver:(RCTPromiseResolveBlock)resolve
                   rejecter:(RCTPromiseRejectBlock)reject) {
     
-    NSData *printData = [NSData dataWithContentsOfFile:filePath];
+    NSData *printData;
+    BOOL isValidURL = NO;
+    NSURL *candidateURL = [NSURL URLWithString:filePath];
+    if (candidateURL && candidateURL.scheme && candidateURL.host)
+        isValidURL = YES;
+
+    if(isValidURL) {
+      // TODO: This needs updated to use NSURLSession dataTaskWithURL:completionHandler:
+      printData = [NSData dataWithContentsOfURL:candidateURL];
+    } else {
+      printData = [NSData dataWithContentsOfFile:filePath];
+    }
```

Comment:
Yep... we had a PDF file we needed to load from a URL.  I was going to make a separate pull request for it, but it looks like I managed to include it on accident, lol... It's working great, though!